cmd/opm/alpha/generate/generate.go

```diff
@@ -0,0 +1,117 @@
+package generate
+
+import (
+	"context"
+	"fmt"
+	"github.com/containerd/containerd/platforms"
+	"github.com/operator-framework/operator-registry/pkg/image"
+	"github.com/operator-framework/operator-registry/pkg/image/containerdregistry"
+	"github.com/operator-framework/operator-registry/pkg/image/containerdregistry/builder"
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "generate <path to declarative config directory>",
+		Short: "Generate an oci-archive index image for a declarative config directory",
+		Args:  cobra.MinimumNArgs(1),
+		RunE: generate,
+	}
+	cmd.Flags().StringP("binary-image", "i", "quay.io/joelanford/opm:a8c80ac9", "container image for on-image `opm` command")
```

Comment:
This is fine for now, but we'll want to drop in the official image before this merges.
```suggestion
	cmd.Flags().StringP("binary-image", "i", "quay.io/operator-framework/upstream-opm-builder:latest", "container image for on-image `opm` command")
```